actions/team_actions.jsx

```diff
@@ -62,14 +62,25 @@ export function removeUserFromTeam(teamId, userId, success, error) {
         (data) => {
             getUser(userId)(dispatch, getState);
             TeamActions.getTeamStats(teamId)(dispatch, getState);
-
-            if (data && success) {
-                success();
+            if (data) {
+                getChannelStats(ChannelStore.getCurrentId())(dispatch, getState).then(
+                    () => {
+                        if (success) {
+                            success();
+                        }
+                    }
+                );
             } else if (data == null && error) {
                 const serverError = getState().requests.teams.removeUserFromTeam.error;
                 error({id: serverError.server_error_id, ...serverError});
             }
         },
+    ).catch(
+        (err) => {
+            if (error) {
+                error(err);
+            }
+        }
```

Comment:
I'm pretty sure this .catch() will never be hit since none of our actions ever throw. Maybe wait until mattermost/mattermost-redux#265 goes in and you can use the [new return pattern](https://pre-release.mattermost.com/core/pl/ap1ue3xtobb6uxssdypq1rudyr) to get the error?